mne/__init__.py

```diff
@@ -98,6 +98,7 @@
 from . import viz
 from . import decoding
 from . import realtime
+from . import testing
```

Comment:
At least `numpy` (and probably other modules) have a `<name>.testing`, which is different from the actual tests, which often live in `<name>.tests`. Thus the motivation for the split. But if you'd prefer not to follow this convention, I can create the `common.py` file, and our imports will be `from mne.tests.common import ...`.